scripts/Healer.java

```diff
@@ -0,0 +1,17 @@
+import nl.tudelft.contextproject.Main;
```

Comment:
I added the scripts in a separate scripts folder, because I kept having to change them during the refactoring, which means decompiling, creating the class in eclipse, compiling with maving, copying the new class into resources and removing the class from eclipse.

Therefore, I thought it would be useful to add the source code for the scripts, to make it easier to find, change and recompile them when necessary.